packages/pixeloven/cli-addon-generators/package.json

```diff
@@ -31,6 +31,7 @@
     "document": "pixeloven-tasks document:ts src",
     "lint": "yarn lint:ts",
     "lint:ts": "pixeloven-tasks lint:ts src/**/*.{ts,tsx}",
+    "postcompile": "copyfiles -u 2 src/templates/**/*.ejs dist/lib/templates",
```

Comment:
This is another thing we could add to the base cli. Gluegun has built ins that could do this with out the need for us to add dependencies

Reply:
@ductiletoaster Interestingly Gluegun's own CLI templates are using the standard `cp` command.

https://github.com/infinitered/gluegun/blob/master/src/cli/templates/cli/package.json.ejs#L18

Do you think it's better to use their `filesystem.copy` or just stick with `cp`?